imageresizerapp/imageresizerapp.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>imageresizerapp</id>
+    <title>Image Resizer for Windows</title>
+    <version>3.0.4802</version>
+    <authors>Brice Lambson</authors>
+    <owners>Anthony Mastrean</owners>
+    <summary>Resize one or more selected image files directly from Windows Explorer.</summary>
+    <description>Image Resizer for Windows is a utility that lets you resize one or more selected image files directly from Windows Explorer by right-clicking. I created it so that modern Windows users could regain the joy they left behind with Microsoft's Image Resizer Powertoy for Windows XP.</description>
+    <projectUrl>http://www.bricelam.net/ImageResizer/</projectUrl>
+    <projectSourceUrl>https://github.com/bricelam/ImageResizer</projectSourceUrl>
+    <packageSourceUrl>https://github.com/AnthonyMastrean/chocolateypackages/</packageSourceUrl>
+    <bugTrackerUrl>https://github.com/bricelam/ImageResizer/issues</bugTrackerUrl>
+    <iconUrl>https://cdn.rawgit.com/AnthonyMastrean/chocolateypackages/c82866c48b2d31da77d7456cfd9f3d32ba0a2b48/public/icons/imageresizerapp.png</iconUrl>
+    <licenseUrl>https://github.com/bricelam/ImageResizer/blob/706812e4ec15ec36f2948a05f9105ac802d35089/LICENSE.md</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
On license acceptance:

The `requireLicenseAcceptance` field comes from the underlying [nuspec technology](http://docs.nuget.org/Create/Nuspec-Reference), but is unused by Chocolatey.

The obligation in using Chocolatey is spelled out in the [Package License Acceptance Terms](https://chocolatey.org/docs/legal#package-license-acceptance-terms).